### PR TITLE
refactor(v2): change the label for docs introduction

### PR DESCRIPTION
**Pull request by fanny**
## Motivation

Rename the docs sidebar label, for me sounds weird the route `docs/docs` in  docs introduction
image

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes
## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)
go to /docs/docs

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: website/docs/docs.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-id: docs
+id: docs-pages-introduction
```
Comment (lex111) [inline website/docs/docs.md:2]: Why not just `docs-introduction`?

Comment (fanny) [reply website/docs/docs.md:2]: it can be :)

**Comment by yangshun** [comment]
This doesn't change the path though, it's still v2.docusaurus.io/docs/docs

**Comment by lex111** [comment]
@yangshun see https://deploy-preview-2716--docusaurus-2.netlify.app/docs/next/docs-introduction

**Comment by yangshun** [comment]
My bad, of course it wouldn't show, I was looking at the stable version.

**Pull request merged**